pyproject.toml

```diff
@@ -152,10 +152,10 @@ tracing = [
     "bentoml[tracing-otlp]",
     "bentoml[tracing-zipkin]",
 ]
-tracing-jaeger = ["opentelemetry-exporter-jaeger==1.14.0"]
-tracing-zipkin = ["opentelemetry-exporter-zipkin==1.14.0"]
-tracing-otlp = ["opentelemetry-exporter-otlp==1.14.0"]
-monitor-otlp = ["opentelemetry-exporter-otlp-proto-http==1.14.0"]
+tracing-jaeger = ["opentelemetry-exporter-jaeger==1.17.0"]
+tracing-zipkin = ["opentelemetry-exporter-zipkin==1.17.0"]
+tracing-otlp = ["opentelemetry-exporter-otlp==1.17.0"]
+monitor-otlp = ["opentelemetry-exporter-otlp-proto-http==1.17.0"]
```

Comment:
I thought we couldn't upgrade this to `1.17.0`

Reply:
I have updated to 1.17 API. The breaking change relies on the `set_logger_provider` import